src/App.jsx

```diff
@@ -1,11 +1,192 @@
 
 import './App.css'
+import {useState}  from "react";
+import Fruitbox from "./components/Fruitbox/Fruitbox.jsx";
+import Button from "./components/Button/Button.jsx";
+import Input from "./components/InputDirectory/Input.jsx";
+
+
 
 function App() {
 
+    const [strawberry, setStrawberry] = useState(0)
+    const [banana,setBanana] = useState(0)
+    const [apple,setApple] = useState(0)
+    const [kiwi,setKiwi] = useState(0)
+
+    const [name,setName] = useState("")
+    const [lastname,setLastName] = useState("")
+    const [age,setAge] = useState(0)
+    const [zipcode,setZipcode] = useState("")
+    const [deliveryFrequency,toggleDeliveryFrequency] = useState("Every-week")
+    const [deliveryTime,toggleDeliveryTime] = useState("EarlyTime")
+    const [comment,setComment] = useState("")
+    const [condition,toggleCondition] = useState(false)
+
+   function resetFruits(){
+        setStrawberry(0)
+        setApple(0)
+        setKiwi(0)
+        setBanana(0)
+    }
+
+    function handleSubmit(e){
+        e.preventDefault()
+        console.log(name,lastname,age,zipcode,deliveryFrequency,deliveryTime,comment,condition)
+        console.log(strawberry,banana,apple,kiwi)
+    }
+
   return (
     <>
-        <h1>Fruitmand bezorgservice</h1>
+        <main>
+            <section className="outer-container">
+                <h1>Fruitmand bezorgservice</h1>
+                    <article className="inner-container">
+                        <p>
+                            <h2>
```

Comment:
Deze drie elementen had je op kunnen nemen in het Fruitbox component, dat had wat code duplicatie gescheeld